projects/com.oracle.truffle.llvm.nativeint/src/com/oracle/truffle/llvm/nativeint/NativeLookup.java

```diff
@@ -93,7 +93,7 @@ public static NativeFunctionInterface getNFI() {
     }
 
     private static List<NativeLibraryHandle> getNativeFunctionHandles() {
-        String[] dynamicLibraryPaths = LLVMBaseOptionFacade.getDynamicLibraryPaths();
+        String[] dynamicLibraryPaths = LLVMOptions.ENGINE.getDYN_LIBRARY_PATHS();
```

Comment:
Could we let the annotation processor generate method names that use Java's naming conventions? It is not a strict requirement, but it would improve readability.